contracts/core/LensHub.sol

```diff
@@ -157,9 +157,11 @@ contract LensHub is ILensHub, LensNFTBase, VersionedInitializable, LensMultiStat
     }
 
     /// @inheritdoc ILensHub
-    function setDefaultProfile(uint256 profileId, address wallet) external override whenNotPaused {
-        _validateCallerIsProfileOwnerOrDispatcher(profileId);
-        _setDefaultProfile(profileId, wallet);
+    function setDefaultProfile(address wallet, uint256 profileId) external override whenNotPaused {
```

Comment:
doesn't this now mean I can unset anyone's default profile anytime even if I don't have the permission to do it? So I could wrap a script to keep unsetting someone's default profile every time they set it? unless I'm missing a check

Reply:
Ah true, missing a check here, fixing.